src/sentry/integrations/slack/notify_action.py

```diff
@@ -74,6 +74,24 @@ class SlackNotifyServiceAction(EventAction):
     form_cls = SlackNotifyServiceForm
     label = u'Send a notification to the {workspace} Slack workspace to {channel} and include tags {tags}'
 
+    def __init__(self, *args, **kwargs):
+        super(SlackNotifyServiceAction, self).__init__(*args, **kwargs)
+        self.form_fields = {
+            'workspace': {
+                'type': 'choice',
+                'choices': [(integration[0], integration[3])
```

Comment:
This might be a bit brittle using field indexes.

I think this can just be written like this

```py
[(i.id, i.name) for i in self.get_integrations()]
```

Is there any reason to use a `value_list`?

Reply:
@EvanPurkhiser thanks, that's way better